packages/kmod-5.15-nvidia/nvidia-fabricmanager.service

```diff
@@ -0,0 +1,18 @@
+[Unit]
+Description=NVIDIA fabric manager service
+After=network-online.target
+Requires=network-online.target
+
+[Service]
+ExecStart=/usr/bin/nvidia-fabricmanager -c /etc/nvidia/fabricmanager.cfg
+Type=simple
+TimeoutSec=0
+RestartSec=5
+Restart=always
+RemainAfterExit=true
```

Comment:
You don't need this line because this isn't a `oneshot`

Reply:
> You don't need this line because this isn't a oneshot

This is a bit of a hack but in combination with `SuccessExitStatus=255` it keeps the service marked as running and started, even when it exits with a failure on instances that don't support Fabric Manager.